{"sections":[],"identifier":{"url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError\/notConnected(_:)","interfaceLanguage":"swift"},"metadata":{"externalID":"s:9VigramSDK21BluetoothServiceErrorO12notConnectedyAC10Foundation4UUIDVcACmF","title":"BluetoothServiceError.notConnected(_:)","role":"symbol","symbolKind":"case","modules":[{"name":"VigramSDK"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"notConnected"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"UUID","preciseIdentifier":"s:10Foundation4UUIDV"},{"kind":"text","text":")"}],"roleHeading":"Case"},"schemaVersion":{"minor":3,"patch":0,"major":0},"abstract":[{"type":"text","text":"Connecting to a device failed unexplicably."}],"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/vigramsdk\/bluetoothserviceerror\/notconnected(_:)"]}],"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"notConnected"},{"kind":"text","text":"("},{"text":"UUID","preciseIdentifier":"s:10Foundation4UUIDV","kind":"typeIdentifier"},{"text":")","kind":"text"}]}]}],"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothService":{"abstract":[{"type":"text","text":"A BluetoothService is used to handle bluetooth connections."}],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothService","role":"symbol","title":"BluetoothService","navigatorTitle":[{"kind":"identifier","text":"BluetoothService"}],"kind":"symbol","type":"topic","url":"\/documentation\/vigramsdk\/bluetoothservice","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"BluetoothService"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothServiceError":{"kind":"symbol","title":"BluetoothServiceError","type":"topic","url":"\/documentation\/vigramsdk\/bluetoothserviceerror","abstract":[{"type":"text","text":"BluetoothServiceError lists different errors that can be propagated by "},{"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothService","type":"reference","isActive":true},{"type":"text","text":" implementations."}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"BluetoothServiceError","kind":"identifier"}],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError","navigatorTitle":[{"text":"BluetoothServiceError","kind":"identifier"}],"role":"symbol"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothServiceError/notConnected(_:)":{"abstract":[{"type":"text","text":"Connecting to a device failed unexplicably."}],"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError\/notConnected(_:)","role":"symbol","title":"BluetoothServiceError.notConnected(_:)","kind":"symbol","type":"topic","url":"\/documentation\/vigramsdk\/bluetoothserviceerror\/notconnected(_:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"notConnected","kind":"identifier"},{"text":"(","kind":"text"},{"text":"UUID","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation4UUIDV"},{"text":")","kind":"text"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","url":"\/documentation\/vigramsdk","title":"VigramSDK","kind":"symbol","role":"collection","images":[{"type":"icon","identifier":"logo.png"}],"type":"topic","abstract":[]},"logo.png":{"variants":[{"traits":["1x","light"],"url":"\/images\/com.vigram.VigramSDK\/logo.png"}],"alt":"Vigram logo.","type":"image","identifier":"logo.png"}}}